src/components/nodes/sequenceFlow/sequenceFlow.vue

```diff
@@ -92,7 +108,7 @@ export default {
     },
     isSourceElementGateway() {
       const sourceShape = this.shape.getSourceElement();
-      return ['bpmn:ExclusiveGateway', 'bpmn:parellelGateway'].includes(sourceShape.component.node.definition.$type);
+      return ['bpmn:ExclusiveGateway', 'bpmn:parellelGateway', 'bpmn:InclusiveGateway'].includes(sourceShape.component.node.definition.$type);
     },
     createLabel() {
       if (!this.node.definition.conditionExpression) {
```

Comment:
This file includes a couple statement of the form:
```js
typeof thing === 'undefined'
```
Since you're touching this file, I think it's worth cleaning up those lines as well:
```js
thing == null
// or
!thing
```